haproxy/README.md

```diff
@@ -56,10 +56,12 @@ $ docker run -it --rm --name haproxy-syntax-check haproxy:1.5 haproxy -c -f /usr
 $ docker run -d --name my-running-haproxy my-haproxy
 ```
 
+You may need to publish the ports your haproxy is listening on to the host by specifying the `-p` option, for example `-p 80:80` to publish port 80. Make sure the port you're using is free.
```

Comment:
Hey @tianon ! Thanks for the suggestion, that's a good one! I think this PR has gone a bit crazy because this note was actually added in a separate PR - https://github.com/docker-library/docs/pull/755 and I'm not sure why/how sneaked in here - sorry about that! So I've updated it according to what you suggested in the other PR :)